data-prepper-plugins/kafka-plugins/src/test/resources/sample-pipelines.yaml

```diff
@@ -25,10 +24,25 @@ log-pipeline:
           retry_backoff: 100  #optional and dafault is 10s
           max_poll_interval: 300000  #optional and dafault is 300000s
           consumer_max_poll_records: 500  #optional and dafault is 500
-        - name: my-topic-1
+        - name: my-topic-2
           group_id: DPKafkaProj-1
       schema:
         registry_url: http://localhost:8081/
         version: 1
+      authentication:
+        sasl_oauth:
+          oauth_client_id: 0oa9wc21447Pc5vsV5d7
+          oauth_client_secret: aGmOfHqIEvBJGDxXAOOcatiE9PvsPgoEePx8IPPa
+          oauth_login_server: https://dev-13650048.okta.com
```

Comment:
As discussed - Most of these should be optional. Did you check in the code after making changes ?

Reply:
Yes, we did it, as per the latest code commit, the following configs are mandatory and rest of them are optional.

1.  oauth_client_id
2.  oauth_client_secret
3.  oauth_login_server
4.  oauth_login_endpoint